src/core/grade_notifier.py

```diff
@@ -251,10 +251,7 @@ def start_notifier():
 
             cursor.execute(f'UPDATE Users SET lastUpdated = NOW() WHERE id={__id};')
             
-            fern = Fernet(os.getenv('DB_ENCRYPTION_KEY').encode('utf-8'))
-            decrypted_password = fern.decrypt(
-                encrypted_password.encode('utf-8')
-            ).decode()
+            decrypted_password = decrypt(encrypted_password, 'test_keys/private.pem')
```

Comment:
We dont want to pass in test_keys, that's just for testing, we want env variables